docs/source/user_guide/server_types.rst

```diff
@@ -0,0 +1,147 @@
+	
+    # instantiate a model
+    from ansys.dpf.core import examples
+    local_model = dpf.Model(examples.find_simple_bar(), server=local_server)
+	
+
+Getting started with DPF GRPC server
+------------------------------------
+
+GRPC communication is enabled using :class:`GrpcServer <ansys.dpf.core.server_types>`. 
+
+.. code-block::
+	   
+    from ansys.dpf import core as dpf
+    grpc_server_config = dpf.AvailableServerConfigs.GrpcServer
+    grpc_server = dpf.start_local_server(config=grpc_server_config)
+    grpc_server
+
+.. rst-class:: sphx-glr-script-out
+
+ .. code-block:: none
+ 
+    <ansys.dpf.core.server_types.GrpcServer object at ...>
+
+You can obtain the server port and ip:
+
+.. code-block::
+
+    print(grpc_server)
+	
+.. rst-class:: sphx-glr-script-out
+
+ .. code-block:: none
+
+    DPF Server: {'server_ip': '127.0.0.1', 'server_port': 50052, 'server_process_id': 9999, 'server_version': '6.0', 'os': 'nt'}
+	
+From a another machine, you can connect remotly to this server and instantiate Models, Operators, and so on:
+
+.. code-block::
+	   
+    from ansys.dpf import core as dpf
+    grpc_remote_server = dpf.connect_to_server(ip='127.0.0.1', port=50052)
+    
+    # instantiate an operator
+    remote_operator = dpf.operators.results.displacement(server=grpc_remote_server)
+    
+    # instantiate a model
+    from ansys.dpf.core import examples
+    remote_model = dpf.Model(examples.find_simple_bar(), server=grpc_remote_server)
+	
+GRPC server use also enables distributed computation capabilities. To learn more about 
+distributed computation with DPF, see :ref:`distributed_post`.
+	
+
+Starting a server using a configuration
+---------------------------------------
+
+The different DPF server types can be started using one of the 
+:class:`AvailableServerConfigs <ansys.dpf.core.server_factory>` configurations. 
+
+.. code-block::
+    
+    in_process_config = dpf.AvailableServerConfigs.InProcessServer
+    in_process_server = dpf.start_local_server(config=in_process_config)
+    
+    grpc_config = dpf.AvailableServerConfigs.GrpcServer
+    grpc_server = dpf.start_local_server(config=grpc_config)
+    
+    legacy_grpc_config = dpf.AvailableServerConfigs.LegacyGrpcServer
+    legacy_grpc_server = dpf.start_local_server(config=legacy_grpc_config)
+
+
+Advanced concepts and release history
+-------------------------------------
+
+The communication logic with a DPF server is defined when starting it using
+an instance of the :class:`ServerConfig <ansys.dpf.core.server_factory>` class.
+Different predefined server configurations are available in DPF,
+each answering a different use-case
+(See the :class:`AvailableServerConfigs <ansys.dpf.core.server_factory>` class).
+
+- The :class:`GrpcServer <ansys.dpf.core.server_types>` configuration is available starting 
+  with server version 4.0 (Ansys 2022 R2).
+  It allows you to remotely connect to a DPF server across a network by telling the client
```

Comment:
Now that I'm reading it again, saying "you" seems strange.
```suggestion
  It allows to remotely connect to a DPF server across a network by telling the client
```

Reply:
No, "it allows you" is correct. Otherwise the sentence is missing a piece. It allows (what?) to remotely connect...